source/extensions/filters/http/common/factory_base.h

```diff
@@ -156,7 +156,7 @@ class DualFactoryBase : public CommonFactoryBase<ConfigProto, RouteConfigProto>,
 
   virtual absl::StatusOr<Envoy::Http::FilterFactoryCb>
   createFilterFactoryFromProtoTyped(const ConfigProto& proto_config,
-                                    const std::string& stats_prefix, DualInfo info,
+                                    const std::string& stats_prefix, DualInfo dual_info,
```

Comment:
this change is not required

Reply:
reverted unintentional change. ty for that pickup.